twill-core/src/main/java/org/apache/twill/internal/AbstractTwillController.java

```diff
@@ -137,7 +138,7 @@ public final synchronized ServiceDiscovered discoverService(String serviceName)
 
   @Override
   public final ListenableFuture<Set<String>> restartInstances(Map<String,
-      ? extends Set<Integer>> runnableToInstanceIds) {
+    ? extends Set<Integer>> runnableToInstanceIds) {
```

Comment:
This new line is awkward, even in the old code. The parent actually returns `Future` instead of returning `ListenableFuture`. If you change the return type to `Future`, everything should fits on one line.